tools/extra/userclk/main.c

```diff
@@ -187,8 +187,7 @@ int main( int argc, char** argv )
 		goto out_destroy_prop;
 	}
 
-	if (userclkCmdLine.freq_high > 0 &&
-		userclkCmdLine.freq_low  > 0) {
```

Comment:
is there no longer a need to check the low frequency?

Reply:
yes
User clock high frequency index used to get values from table